lib/chore/duplicate_detector.rb

```diff
@@ -23,7 +14,21 @@ def initialize(opts={})
       @dupe_on_cache_failure = opts.fetch(:dupe_on_cache_failure) { false }
       @timeout               = opts.fetch(:timeout) { 0 }
       @servers               = opts.fetch(:servers) { nil }
-      @memcached_client      = opts.fetch(:memcached_client) { Dalli::Client.new(@servers, memcached_options) }
+      @memcached_client      = opts.fetch(:memcached_client) { memcached_client(memcached_options) }
```

Comment:
Strictly speaking this is already being set by making the call but it does make the code line up nice.